http3/server.go

```diff
@@ -690,16 +685,14 @@ func ListenAndServeQUIC(addr, certFile, keyFile string, handler http.Handler) er
 // The correct Alt-Svc headers for QUIC are set.
 func ListenAndServe(addr, certFile, keyFile string, handler http.Handler) error {
 	// Load certs
-	var err error
-	certs := make([]tls.Certificate, 1)
-	certs[0], err = tls.LoadX509KeyPair(certFile, keyFile)
+	cert, err := tls.LoadX509KeyPair(certFile, keyFile)
```

Comment:
changed to not allocate the slice with a make before we are sure we don't have any errors 
and if all ok no need for the make we can directly add to the tls.Config struct